### ISSUE TITLE
bRING THE OLD ONE BACK

### ISSUE THREAD

**Issue by vivpet** [CaliAlec/ChromeIGStory#113]
I HATE the new update, It doesnt  play all automatically . I cant download, it really sucks, looking for another app!

**Comment by ricardokdz** [comment]
+1

**Comment by adamjapon** [comment]
Yeah... I much prefer the version before the update... Where you can see the time stamp of the last post.
And it automatically playable to the next stories. Thus one will go gray after we saw it. 
Vote for the old version! Hope @CaliAlec  reading this... ;)
